handbook/company/pricing-features-table.yml

```diff
@@ -447,7 +448,9 @@
       tier: Free
       usualDepartment: IT
       waysToUse:
-        - description: Deploy custom configuration profiles on macOS and Windows (coming soon) and verify that they're installed.
+        - description: Deploy custom profiles on macOS and verify that they're installed. Windows coming soon (2023-12-31). #13281
+        - description: Deploy custom declaration (DDM) profiles on macOS. Coming soon (2024-03-31). #14550
+        - description: Target profiles to specific hosts using SQL. Coming soon (2023-12-31) #14715
```

Comment:
Mike: "using SQL" doesn't sound easy in an ad.

Reply:
I agree. We're using it elsewhere: https://github.com/fleetdm/fleet/pull/14934/files#diff-f109cc3f7421e05eddf317b7c6cc51395977e63a25dd9eed437985060ed796edL528

To move quickly, let's address this later.